### PR TITLE
Docker compose configuration for running Identity Server with Analytics

### PR DESCRIPTION
**Pull request by msmshariq**
## Purpose
Since docker compose is mainly used to tryout the product, we no longer consider implementing pattern as part of docker compose. Instead we will provide simplified docker compose configs that doesn't consider HA / Clustering etc. 

As part of this initiative, this PR provides a docker compose configuration that runs an `Identity Server container` configured to publish stats and an `Identity Server Analytics container` that will process the published events and display analytics dashboards. 

## Goals
Simplify the docker compose configurations and remove the pattern dirs.

### CODE REVIEW

File: docker-compose/is-analytics/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+     ```
+
+  3. Download the latest Identity Server Docker resources release zip file from the [releases](https://github.com/wso2/docker-is/releases) page or clone this repository and switch to the latest tag.
+
+  4. Switch to the docker-compose/is-analytics folder:
+     ```
+     cd [docker-is]/docker-compose/is-analytics
+     ```
+
+  5. Download [MySQL Connector/J](https://downloads.mysql.com/archives/c-j/) v5.1.35 and copy its JAR file to the following path:
```
Comment (deki) [inline docker-compose/is-analytics/README.md]: Link is outdated, should be https://dev.mysql.com/downloads/connector/j/

File: docker-compose/is-analytics/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+     ```
+     cp path/to/mysql-connector-java-5.1.35/mysql-connector-java-5.1.35-bin.jar [docker-is]/docker-compose/is-analytics/identity-server/repository/components/lib/
```
Comment (deki) [inline docker-compose/is-analytics/README.md]: Please do not use old 5.1.35 version, it has security issues: https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-3523

File: docker-compose/is-analytics/README.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
+# WSO2 Identity Server with Analytics
+
+Runs a pre configured Identity Server container and Identity Server Analytics container.
+
+## Prerequisites
+
+ * [Docker](https://www.docker.com/get-docker) and [Docker Compose](https://docs.docker.com/compose/install/#install-compose) are required for running this Docker Compose template.
```
[the remark this answers is not in the document]
Comment (chirangaalwis) [reply docker-compose/is-analytics/README.md]: +1

File: docker-compose/is-analytics/README.md (hunk 1 above)
Comment (DilanUA) [reply docker-compose/is-analytics/README.md]: @deki, Thanks for pointing out this. Will do a platform wide update.

File: docker-compose/is-analytics/README.md (hunk 2 above)
Comment (msmshariq) [reply docker-compose/is-analytics/README.md]: @deki - I've created issue #29 to update the JDBC driver version. Will fix this as a separate PR.

**Pull request merged**